libraries/psibase/common/include/psibase/RawNativeFunctions.hpp

```diff
@@ -537,6 +561,46 @@ namespace psibase
       raw::writeConsole(sv.data(), sv.size());
    }
 
+   using raw::abortSubjective;
+   using raw::checkoutSubjective;
+   using raw::commitSubjective;
+
+   struct SubjectiveTransaction
+   {
+      SubjectiveTransaction() { psibase::checkoutSubjective(); }
+      ~SubjectiveTransaction()
+      {
+         if (!done)
+         {
+            psibase::abortSubjective();
+         }
+      }
+      void commit()
+      {
+         if (!done)
+         {
+            done = psibase::commitSubjective();
+         }
+      }
+      bool done = false;
+   };
+
+   /// The `PSIBASE_SUBJECTIVE_TX` macro creates a scope in which
```

Comment:
Did you intend for this comment to be visible in the mdbook docs? If so, I'm not seeing it

Reply:
I tried, but libclang doesn't provide a way to get the comment.